OPHD/States/PlanetSelectState.cpp

```diff
@@ -86,6 +88,13 @@ PlanetSelectState::~PlanetSelectState()
 	Utility<Mixer>::get().stopAllAudio();
 }
 
+namespace {
+	std::vector<Planet::Attributes> planetAttributes = {
```

Comment:
For hardcoded data, you could potentially use a `std::array` here, since it's a fixed size. Long term though, if data is eventually loaded from a file, the `std::vector` makes more sense. I assume that's what you were going for.